core/src/test/java/edu/wpi/grip/core/CoreSanityTest.java

```diff
@@ -27,7 +27,8 @@ public CoreSanityTest() {
         ManualPipelineRunner.class,
         SubtractionOperation.class,
         Main.class,
-        CoreCommandLineHelper.class
+        CoreCommandLineHelper.class,
+        Pipeline.class
```

Comment:
That makes sense. Uhhh.
In the code below this where all the `setDefault`'s are add:
```java
setDefault(Step.class, new MockStep());
```
That should solve this problem.